packages/core/lib/send-result/node.ts

```diff
@@ -0,0 +1,93 @@
+import type { ServerResponse } from "http";
+import type { Http2ServerResponse } from "http2";
+import { HttpError } from "../errors";
+import type {
+  Response,
+  MultipartResponse,
+  Push,
+  ProcessRequestResult,
+} from "../types";
+
+type RawResponse = ServerResponse | Http2ServerResponse;
+
+export async function sendResponseResult(
+  responseResult: Response<any, any>,
+  rawResponse: RawResponse
+) {
+  rawResponse.statusCode = responseResult.status;
+  rawResponse.end(JSON.stringify(responseResult.payload));
+}
+
+export async function sendMultipartResponseResult(
+  multipartResult: MultipartResponse<any, any>,
+  rawResponse: RawResponse
+) {
+  rawResponse.writeHead(200, {
+    // prettier-ignore
+    "Connection": "keep-alive",
+    "Content-Type": 'multipart/mixed; boundary="-"',
+    "Transfer-Encoding": "chunked",
+  });
+
+  rawResponse.on("close", () => {
+    multipartResult.unsubscribe();
+  });
+  // @ts-ignore - Different Signature between ServerResponse and Http2ServerResponse but still compatible.
```

Comment:
shouldn't these be @ts-expect-error ?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error

Not sure actually, I used ts-expect-error in the context of envelop before but it caused issues when supporting multiple GraphQL.js issues (as in one version it was an error and in the other it wasnt.

I guess in this context both do the same job

Reply:
I changed it in hope that there might be no conflict in the @types/node package one day